### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by mkarlstrand-splunk**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -20,19 +20,19 @@ Post a question to Splunk Answers using the tag "Splunk Connect For Syslog"
...
 
-Please use the GitHub issue tracker to submit bugs or request enhancements: https://github.com/splunk/splunk-connect-for-syslog/issues
+Please use the GitHub issue tracker to submit bugs or request enhancements: [Issues](https://github.com/splunk/splunk-connect-for-syslog/issues)
 
 Get involved, try it out, ask questions, contribute filters, and make new friends!
 
 ## Contributing
 
-We welcome feedback and contributions from the community! Please see our [contribution guidelines](CONTRIBUTING.md) for more information on how to get involved.
+We welcome feedback and contributions from the community! Please see our [contribution guidelines](https://github.com/splunk/splunk-connect-for-syslog/blob/master/CONTRIBUTING.md) for more information on how to get involved.
```
Comment (rfaircloth-splunk) [inline docs/index.md]: the original links were relative and should be kept not the long form URL

File: docs/index.md [hunk 2]
```diff
@@ -20,19 +20,19 @@ Post a question to Splunk Answers using the tag "Splunk Connect For Syslog"
...
 
 ## License
 
-* Configuration and documentation licensed subject to [CC0](LICENSE-CC0)
+* Configuration and documentation licensed subject to [CC0](https://github.com/splunk/splunk-connect-for-syslog/blob/master/LICENSE-CC0)
```
Comment (rfaircloth-splunk) [inline docs/index.md]: same as above

File: docs/index.md [hunk 3]
```diff
@@ -20,19 +20,19 @@ Post a question to Splunk Answers using the tag "Splunk Connect For Syslog"
...
 
-* Code and scripts licensed subject to [BSD-2-Clause](LICENSE-BSD2) 
+* Code and scripts licensed subject to [BSD-2-Clause](https://github.com/splunk/splunk-connect-for-syslog/blob/master/LICENSE-BSD2)
```
Comment (rfaircloth-splunk) [inline docs/index.md]: same as above

File: docs/index.md (hunk 1 above)
Comment (mkarlstrand-splunk) [reply docs/index.md]: As is it returns a 404. What is a better solution than the full URL link?

File: docs/index.md (hunk 2 above)
Comment (mkarlstrand-splunk) [reply docs/index.md]: Yes, same as above, 404.

File: docs/index.md (hunk 3 above)
Comment (mkarlstrand-splunk) [reply docs/index.md]: Yes

File: docs/index.md (hunk 1 above)
Comment (rfaircloth-splunk) [reply docs/index.md]: oh its in the root we need to keep a copy of that file copied down into the docs folder

File: docs/index.md (hunk 2 above)
Comment (rfaircloth-splunk) [reply docs/index.md]: same we should keep a copy of this file copied into the docs folder

**Comment by rfaircloth-splunk** [comment]
Closed replaced by another

**Pull request closed without merging**